<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qGUIProjectClass</class>
 <widget class="QMainWindow" name="qGUIProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qGUIProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnShowVids">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Alteration</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rbAlter1">
         <property name="text">
          <string>High FPS Window</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbAlter2">
         <property name="text">
          <string>Low FPS Window</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Drawing Framework</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rbFramework1">
         <property name="text">
          <string>OpenCV</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbFramework2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>QT</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnTerminate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Terminate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qguiproject.qrc"/>
 </resources>
 <connections/>
</ui>
